resources/views/components/manage-notifications.blade.php

```diff
@@ -194,7 +194,8 @@ class="cursor-pointer text-theme-secondary-300 hover:text-theme-primary-500 focu
                                     </div>
                                     <div class="leading-5 text-theme-secondary-600">
                                         <div class="flex flex-col sm:block">
-                                            <span class="break-words">{{ $notification->excerpt() }}</span>
```

Comment:
why remove `break-words`? now large words/urls will end up going out of bounds again which is what the card was about